src/contents/articles/hello-world/index.md

```diff
@@ -27,4 +25,4 @@ We want to grow as software developers but we also want to grow as a people. As
 
 ## Join us!
 
-We got a whole lot of things planned out for you, and we really believe we'll all feel better about our profession and passion if we do it together. With namespace, local developers and enthusiasts will finally have that common ground - one module, no globals.
```

Comment:
"We'll all feel better" is not a compelling call to action. I think you need to be explicit about how participating in a community will be beneficial to prospective members/attendees.